usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -4995,6 +4993,24 @@ lxpr_read_sys_kernel_rand_entavl(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	lxpr_uiobuf_printf(uiobuf, "%d\n", swrand_stats.ss_entEst);
 }
 
+static void
+lxpr_read_sys_kernel_rand_uuid(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
+{
+	/*
+	 * Each read from this read-only file should return a new
+	 * random 128-bit UUID string in the standard UUID format.
+	 */
+	zone_t *zone = LXPTOZ(lxpnp);
+	char uuid[LX_BOOTID_LEN];
+
+	ASSERT(lxpnp->lxpr_type == LXPR_SYS_KERNEL_RAND_UUID);
+	ASSERT(zone->zone_brand == &lx_brand);
```

Comment:
In smartos lx proc can be found at `/system/lxproc` in a native zone.  I don't see the sys dir under that path, but I do see it in an lx zone.  I just want to confirm someone in a native zone cant use this path to trigger this assert. I would like to verify that we don't mount sys in native.

Reply:
https://github.com/joyent/illumos-joyent/blob/2a7c46f8d5561b2947bce41d61abe7aae999ba8e/usr/src/uts/common/brand/lx/procfs/lx_prvnops.c#L31

I guess this answers my question :)